commands/Tools/contact.js

```diff
@@ -0,0 +1,40 @@
+exports.run = async (client, msg, [type, ...content]) => {
+  try {
+    // Determines which type of contact and sends to the appropriate channel.
+    if (type === "bug") {
+      this.channel.send(`Bug Report From: ${msg.author} in ${msg.guild}\n\n ${content.toString().replace(/,/g, " ")}`);
```

Comment:
Reduce the code repetition and redundancy with a single `this.channel.send`. Another thing I have noticed, this command runs in any channel, even in DMs, `msg.guild` will output `null` in the latter case.